### PR TITLE
Update group_seven.md

### PR DESCRIPTION
**Pull request by blackknight75**

### CODE REVIEW

**Comment by blackknight75** [comment]
@emhickmann

File: m4_sessions/1705-inning/group_seven.md [hunk 1]
```diff
@@ -2,9 +2,46 @@
...
-### Topic: Ethics and Technology of the Snowden Leaks
+# Ethicas and Technology of the Snowden Leaks
+> 
 
-### Outline
-#### Before class: Readings before session (short, infographic-like) about technologies revealed in Snowden documents
-#### Video: debate snippets showing both sides
-#### Discussion: Ask questions for pair and share discussion
+## Pre-Material
+Please review these resources to prepare you for our session. Think about what is important to you and why.
```
Comment (emhickmann) [inline m4_sessions/1705-inning/group_seven.md:9]: Make sure you send this out by Wednesday evening to give people time!

File: m4_sessions/1705-inning/group_seven.md [hunk 2]
```diff
@@ -2,9 +2,46 @@
...
+#### Prism From Edward Snowden
+>https://www.youtube.com/watch?v=3P_0iaCgKLk
+
+#### What is XKeyScore
+> https://www.youtube.com/watch?v=3IzXkLOgVZ8
+
+#### All of Snowden's Files (Optional)
+> https://snowdenarchive.cjfe.org/greenstone/cgi-bin/library.cgi?e=p-00100-00---off-0snowden1--00-2----0-10-0---0---0direct-10----4-------0-1l--10-en-50---50-about---01-3-1-00-00--4--0--0-0-01-10-0utfZz-8-00&a=d&cl=CL6
+
+## Reading/Video (15 minutes )
```
Comment (emhickmann) [inline m4_sessions/1705-inning/group_seven.md:23]: Make sure that you have each of the links ready to go and AV tested

File: m4_sessions/1705-inning/group_seven.md [hunk 3]
```diff
@@ -2,9 +2,46 @@
...
+We will be watching 2 videos, 1 on each side of the argument. 
+> VIDEO LINK 1
+> VIDEO LINK 2
+> READING LINK
+
+## Reflection ( 10 Minutes )
```
Comment (emhickmann) [inline m4_sessions/1705-inning/group_seven.md:29]: I might give a few less minutes for reflection (5) and add more to the discussion

File: m4_sessions/1705-inning/group_seven.md [hunk 4]
```diff
@@ -2,9 +2,46 @@
...
+> What are your thoughts on the ethics of Snowden leaking this technology?
+
+
+> Question 3 (How this effects you??)
+> In his position what would you do and why? Is it ok to leak our technology?
+
+> Question 4 (Takeaways)
+
+## Group Share ( 10 minutes )
+> Share your reflections and opinions and come up with group pros and cons
```
Comment (emhickmann) [inline m4_sessions/1705-inning/group_seven.md:44]: Determine how you want people to group up.  Maybe even suggest 1 or two specific questions (2 and 3) for the group discussion.

File: m4_sessions/1705-inning/group_seven.md [hunk 5]
```diff
@@ -2,9 +2,46 @@
...
+
+## Share Back to Full Group ( 10 minutes )
```
Comment (emhickmann) [inline m4_sessions/1705-inning/group_seven.md:46]: Think about what you want to close with...is there a question you want everyone to consider?  More resources?  A follow up slack channel to continue the convo?

**Pull request merged**